app/routes/document.js

```diff
@@ -41,6 +43,8 @@ export default class DocumentRoute extends Route {
 
     const params = this.paramsFor(this.routeName);
     this.isSigning = params.tab === this.intl.t('signatures');
+    const shortPieceName = new VRDocumentName(model?.name).vrNumberWithSuffix();
+    setHash(shortPieceName);
```

Comment:
Problem here is that there are multiple types of documents, some have their own VR numbering parsing
![image](https://github.com/kanselarij-vlaanderen/frontend-kaleidos/assets/22245223/3261ffd0-1681-4d7c-a712-8175d98df288)

Reply:
so in many cases we would fall back to just the full name, which might not be desired?
example `VR PV 2024/7 - ALLE BESLISSINGEN` which actually makes sense, because this is a different file than `VR PV 2024/7`

Reply:
AFAIK this feature/ticket is meant to enable the document stamping to be "visible" in the UI *withouth* being stamped, so that we only need to stamp when downloading/printing.

I.e. this doesn't really matter for beslissingsfiches, notulen, etc. so we might even want to disable this for those document types.

Reply:
So, we'll leave this as-is. We try to parse it using the VR doc name parser. If that doesn't work, we show the full name.
Should be ok. We always show something normally.